elide-core/src/main/java/com/yahoo/elide/core/EntityBinding.java

```diff
@@ -219,19 +219,19 @@ private void bindAttrOrRelation(Class<?> cls, AccessibleObject fieldOrMethod) {
             RelationshipType type;
             String mappedBy;
             if (oneToMany) {
-                type = RelationshipType.ONE_TO_MANY;
+                type = computedRelationship ? RelationshipType.COMPUTED_ONE_TO_MANY : RelationshipType.ONE_TO_MANY;
```

Comment:
Just to verify: using the existing `@ManyToMany`, `@OneToOne`, etc. annotations will require any compound models to be _distinct_ from their persistent representations. Otherwise, even though we claim this is a computed attribute, hibernate schema generation and similar will pick these up as concrete fields.